src/Auth0.OidcClient.Android/AndroidWebViewBase.cs

```diff
@@ -0,0 +1,40 @@
+﻿using IdentityModel.OidcClient.Browser;
+using System;
+using System.Threading.Tasks;
+
+namespace Auth0.OidcClient
+{
+    public abstract class AndroidWebViewBase : IBrowser
+    {
+        public Task<BrowserResult> InvokeAsync(BrowserOptions options)
+        {
+            if (string.IsNullOrWhiteSpace(options.StartUrl))
+                throw new ArgumentException("Missing StartUrl", nameof(options));
+
+            if (string.IsNullOrWhiteSpace(options.EndUrl))
+                throw new ArgumentException("Missing EndUrl", nameof(options));
+
+            var tcs = new TaskCompletionSource<BrowserResult>();
+
+            void Callback(string response)
+            {
+                ActivityMediator.Instance.ActivityMessageReceived -= Callback;
+
+                var cancelled = response == "UserCancel";
```

Comment:
should this be part of a constant? here you're comparing, but where is it set originally?

Reply:
It's set by the OIDC IdentityModel internally.

Reply:
It's set in the ActivityMediator.